File: go/calc.go
package go_calc

func add(a, b float64) float64 { return a + b }
func sub(a, b float64) float64 { return a - b }
func mul(a, b float64) float64 { return a* b }
func div(a, b float64) float64 {
	if b == 0 {
		panic("division by zero")
	}
	return a / b
}

File: php/calc.php
<?php

function add($a, $b) { return $a + $b; } // addition
function sub($a, $b) { return $a - $b; } // subtraction
function mul($a, $b) { return $a * $b; } // multiplication
function div($a, $b) {
    if ($b == 0) {
        throw new DivisionByZeroError("division by zero");
    }
    return $a / $b;
}